src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java

```diff
@@ -36,9 +39,29 @@ public CftQueryService(TaskResourceRepository taskResourceRepository) {
         Sort sort = SortQuery.sortByFields(searchTaskRequest);
 
         Pageable page = PageRequest.of(firstResult, maxResults, sort);
-        final Page<TaskResource> taskResources = taskResourceRepository.findAll(taskResourceSpecification, page);
+        final Page<TaskResource> pages = taskResourceRepository.findAll(taskResourceSpecification, page);
 
-        return taskResources.toList();
+        final List<TaskResource> taskResources = pages.toList();
+
+        return mapToTask(taskResources, pages.getTotalElements());
+    }
+
+    private GetTasksResponse<Task> mapToTask(List<TaskResource> taskResources, long totalNumberOfTasks) {
```

Comment:
Same thing here, why not keep the standard list and wrap the answer in GetTasksResponse in the controller

Reply:
I got it. will change.